### ISSUE TITLE
It seems the macro ignores `prefix`

### ISSUE THREAD

**Issue by nguiard** [SeriousBug/rust-embed-for-web#10]
Hi! Thanks a lot for this nice crate!

I'm playing around with it and it seems that your `RustEmbed` macro ignores the `prefix`. 
Currenty I have an `assets` folder with a css subfolder and files, and:

```rust
#[derive(RustEmbed)]
#[folder = "assets"]
#[prefix = "foo/bar"]
struct StaticAssets;
```
and later:
```rust
StaticAssets::get("css/tw.css"); // Some
StaticAssets::get("foo/bar/css/tw.css"); // None
```

Am I doing something wrong?

**Comment by SeriousBug** [comment]
Hey, thanks for checking it out!

Prefix support is not implemented at all. It looks like I accidentally left [the proc macro definition](https://github.com/SeriousBug/rust-embed-for-web/blob/be2bb8c2b6275430b38c656000a06e82ecdb4886/impl/src/lib.rs#L81), but it's not actually implemented.

I'm happy to accept a PR to implement this if you'd like, I can point towards where to start. Otherwise I can probably work on this to add support this weekend.

**Comment by nguiard** [comment]
Hey, I'd be happy to have a try at it :) if you have some pointers to start they're welcome.

**Comment by nguiard** [comment]
PR is ready! #11

**Comment by SeriousBug** [comment]
Closed by #11
